app.js

```diff
@@ -0,0 +1,57 @@
+const axios = require('axios');
+const express = require('express')
+const app = express()
+const port = process.env.PORT || 3002;
+
+
+const url = "https://swapi.dev/api/"
+
+
+app.get('/:entyds/:id', async (req, res) => {
+    console.time('run>>')
+    const { entyds, id } = req.params;
+    
+    const urlResquest = `${url}/${entyds}/${id}`;
+    const enrichFieldsParams = req.query.enrichFields;
+
+    const filmResponse = await axios.get(urlResquest);
+    const film = filmResponse.data;
+    
+    if (enrichFieldsParams !== undefined) {
+        
+        const enrichFields = enrichFieldsParams.split(",")
+        
+        if (filmResponse.status === 200) {
+            for (let field of enrichFields) {
+                
+                const currentField = film[field];
+                
+                const fullFields = [];
+                
+                for (const url of currentField) {
+                    
+                    const responseField = await axios.get(url);
```

Comment:
Por causa do for + await o seu algoritmo está fazendo todos os requests de maneira síncrona, ou seja, um após o outro. 

Isso remove a característica mais importante de concorrência e velocidade do JS, que é capaz de fazer múltiplos downloads assíncronos de forma concorrente. 

Dá uma lida em: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all